opentext-page.html

```diff
@@ -0,0 +1,482 @@
+                        <li><a href="#Consulting">Consulting</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="column">
+                <div class="row">
+                    <div a href="#Resources">Resources</a></div>
+                    <ul>
+                        <li><a href="#Voyage">Voyager</a></li>
+                        <li><a href="#Events">Events & webinars</a></li>
+                        <li><a href="#Blogs">Blogs</a></li>
+                        <li><a href="#Press">Press releases</a></li>
+                        <li><a href="#Customer">Customer stories</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="slide-show">
+            <!-- div has two columns will add second column when learn about slideshows-->
+            <img src="#img1" alt="banner for voyager fund">
+            <h2>OpenText launches Voyager fund</h2>
+            <p>1M donation to help fight food insecurity</p>
+            <button type="submit">Learn more</button>    
+        </div>
+        <div class="slide-show">
+             <!-- div has two columns will add second column when learn about slideshows -->
+            <img src="#img2" alt="banner for cloud">
+            <h2>Unleash the power of cloud</h2>
+            <p>Feauturing Guest Speaker</p>
+            <p>Frank Dellla Rosa</p>
+            <p>Researcher Director, IDC</p>
+            <button type="submit">Watch the webinar</button>
+        </div>
+        <div class="slide-show">
+             <!-- div has two columns willl add second column when learn about slideshows -->
+            <img src="#img3" alt="image of astronaut">
+            <h2>Defy gravity</h2>
+            <p>Rethink the complexity of your supply chain operations</p>        
+            <button type="submit">Learn more</button>
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Cloud login</span>
+                </div>
+            </div>
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Get Support</span>
+                </div>
+            </div>
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Developer</span>
+                </div>
+            </div>
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Contact us</span>
```

Comment:
same issue as above, change this across the board for class names...
should never be their element name